### PR TITLE
v2 sdk changes emr

### PR DESCRIPTION
**Pull request by Trianz-Akshay**
*Issue #, if available:*
#2086 SDK v2 changes for EMR service

*Description of changes:*
SDK v2 changes for EMR service

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: athena-aws-cmdb/src/main/java/com/amazonaws/athena/connectors/aws/cmdb/TableProviderFactory.java [hunk 1]
```diff
@@ -59,14 +59,14 @@ public TableProviderFactory(java.util.Map<String, String> configOptions)
     {
         this(
             AmazonEC2ClientBuilder.standard().build(),
-            AmazonElasticMapReduceClientBuilder.standard().build(),
+            EmrClient.builder().region(DefaultAwsRegionProviderChain.builder().build().getRegion()).build(),
```
Comment (aimethed) [inline athena-aws-cmdb/src/main/java/com/amazonaws/athena/connectors/aws/cmdb/TableProviderFactory.java:62]: Use `EmrClient.create()`. It uses the default aws region provider chain.

File: athena-hbase/src/test/java/com/amazonaws/athena/connectors/hbase/integ/HbaseIntegTest.java [hunk 2]
```diff
@@ -179,27 +179,27 @@ private Pair<App, Stack> getHbaseStack() {
      */
     private String getClusterData()
     {
-        AmazonElasticMapReduce emrClient = AmazonElasticMapReduceClientBuilder.defaultClient();
+        EmrClient emrClient = EmrClient.builder().region(DefaultAwsRegionProviderChain.builder().build().getRegion()).build();
```
Comment (aimethed) [inline athena-hbase/src/test/java/com/amazonaws/athena/connectors/hbase/integ/HbaseIntegTest.java:182]: Same as other. `EmrClient.create()`

**Review by aimethed — CHANGES_REQUESTED** [review]

File: athena-aws-cmdb/src/main/java/com/amazonaws/athena/connectors/aws/cmdb/TableProviderFactory.java (hunk 1 above)
Comment (Trianz-Akshay) [reply athena-aws-cmdb/src/main/java/com/amazonaws/athena/connectors/aws/cmdb/TableProviderFactory.java]: updated with the changes suggested.

File: athena-hbase/src/test/java/com/amazonaws/athena/connectors/hbase/integ/HbaseIntegTest.java (hunk 2 above)
Comment (Trianz-Akshay) [reply athena-hbase/src/test/java/com/amazonaws/athena/connectors/hbase/integ/HbaseIntegTest.java]: updated with the changes suggested.

**Pull request merged**